packages/adapter/addon/index.ts

```diff
@@ -321,7 +316,6 @@ export default EmberObject.extend({
     @param {Snapshot} snapshot
     @return {Promise} promise
   */
-  createRecord: null,
```

Comment:
I removed this because this should be typed as a function. Looking at moving these to `ts` in this PR or another.